srcpkgs/kafkacat/template

```diff
@@ -1,7 +1,8 @@
 # Template file for 'kafkacat'
 pkgname=kafkacat
 version=1.6.0
-revision=1
+revision=2
```

Comment:
Looks like this package changed out from under us. (different checksum, different `wrksrc`), but it *does* bulid and run. 

Not sure how we handle situations like this.